### DISCUSSION TITLE
Termux pkg install error

### DISCUSSION THREAD

**Discussion by iceCream-boop** [termux/termux-packages#7334] [Q&A]

**Comment by ghost** [comment]
Do you mean `pkg` tool itself or referring to problems with installing specific ~~pkg~~ package? What error did you get?

Please ask the question correctly. If you have a problem with using certain tool, explain it.

Solution for repository issues is provided there: https://github.com/termux/termux-packages/issues/6726

**Reply by deva117** [reply]
Me also facing trouble in updating termux pkgs. And i can't install any othertools

**Reply by ghost** [reply]
\-\-\> https://github.com/termux/termux-packages/issues/6726

**Comment by BhupeshBhatia48** [comment]
What to do it's been 3 months the repository is still missing how we are able to install any type of pakage I love termux a lot please solve this issue

**Reply by ghost** [reply]
Use Termux from F-Droid, coz Play Store version is barely dead. If decided to switch, please open Android settings --> Applications and uninstall Termux, Termux:API, Termux:Boot, Termux:Float, Termux:Styling, Termux:Task, Termux:Widget before installing app from F-Droid.

F-Droid: https://f-droid.org/en/packages/com.termux/

Issue: https://github.com/termux/termux-packages/issues/6726 - Bintray repositories are down because service no longer exist. As we do not own Bintray or its domain names, we can't do anything. Just a reminder that no online services last forever.
